python/lsst/daf/butler/script/exportCalibs.py

```diff
@@ -37,20 +38,20 @@
 from ..registry import CollectionType
 
 if TYPE_CHECKING:
-    from lsst.daf.butler import DatasetRef, DatasetType, Registry
+    from lsst.daf.butler import DatasetRef, DatasetType
 
 log = logging.getLogger(__name__)
 
 
 def parseCalibrationCollection(
-    registry: Registry, collection: str, datasetTypes: Iterable[DatasetType]
-) -> tuple[list[str], list[DatasetRef]]:
+    butler: Butler, collection: str, datasetTypes: Iterable[DatasetType]
+) -> tuple[set[str], list[DatasetRef]]:
```

Comment:
Might be a good time to rename this function, since it's not really doing any "parsing".